Dockerfile

```diff
@@ -18,9 +18,9 @@ RUN apt-get update && apt-get -y install  unzip \
                         pkg-config \
                         p7zip-full
 
-ENV KERNEL_VERSION  3.16.7
-ENV AUFS_BRANCH     aufs3.16
-ENV AUFS_COMMIT     b3883c3cb86937801fdd2e188032063de617ef73
+ENV KERNEL_VERSION  3.18.1
+ENV AUFS_BRANCH     aufs3.18
```

Comment:
Reading mailing lists upstream, it looks like they switched to the `aufs3.18.1+` branch for the 3.18 development.  Does it make sense for us to switch too?